File: packages/api/cmd/sikuligrpc/examples/node/workflow-connect.mjs
import { ensureSikuligoOnPath } from "./bootstrap.mjs";
import { Screen, Pattern } from "@sikuligo/sikuligo";

ensureSikuligoOnPath();

// Connect-only workflow (requires sikuligo already running).
const screen = await Screen.connect();
try {
  const match = await screen.click(Pattern("assets/pattern.png").exact());
  console.log(`clicked match target at (${match.targetX}, ${match.targetY})`);
} finally {
  await screen.close();
}

File: packages/api/cmd/sikuligrpc/examples/python/workflow-connect.py
from __future__ import annotations

from bootstrap import ensure_sikuligo_on_path
from sikuligo import Pattern, Screen

ensure_sikuligo_on_path()

# Connect-only workflow (requires sikuligo already running).
screen = Screen.connect()
try:
    match = screen.click(Pattern("assets/pattern.png").exact())
    print(f"clicked match target at ({match.target_x}, {match.target_y})")
finally:
    screen.close()
